schema/codelists/directOrIndirect.csv

```diff
@@ -1,4 +1,4 @@
 code,title,description,technical note
-direct,Direct,The interest is held directly.,
-indirect,Indirect,The interest is held through one or more intermediate entities (including arrangements).,
-unknown,Unknown,The interest may be direct or indirect.,
+direct,Direct,"The interest is held directly.",
+indirect,Indirect,"The interest is held through one or more intermediate entities (including arrangements).",
+unknown,Unknown,"The interest may be direct or indirect.",
```

Comment:
Would it be worth saying 'or both' - I'm not sure. We show this in some of the examples where ownership is direct and indirect.

Reply:
I'm going to leave as-is since....

The question of both-ness touches on how we probably need to develop the Interest modelling ... see https://github.com/openownership/data-standard/issues/551 .